build/README.md

```diff
@@ -0,0 +1 @@
+This is the directory where the static site gets built.
```

Comment:
all these oneliner readmes are there just so the folders get commited 😜 this means we don't have to mkdir every time